src/hci.cpp

```diff
@@ -4386,6 +4386,54 @@ void stopReticuleButtonFlash(UDWORD buttonID)
 	}
 }
 
+// show seected widget from reticule menu
```

Comment:
seected -> selected

Reply:
Typos has been fixed, about 282-287 - this is example, not included in rules.js in this commit, after accept this commit, i have some write more correct logic in rules.js and prepare another pull request.
What would the commits were separate cpp from js.